components/string.rst

```diff
@@ -125,7 +125,7 @@ There are also some specialized constructors::
     $foo = ByteString::fromRandom(12);
 
     // CodePointString and UnicodeString can create a string from code points
-    $foo = UnicodeString::fromCodePoints(0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947);
+    $foo = (string) UnicodeString::fromCodePoints(0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947);
```

Comment:
I think you are right from the technical point of view, but I guess this should only show, what `0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947` will result in 🤔 

cc @nicolas-grekas